src/helpers/issue.ts

```diff
@@ -95,6 +95,22 @@ export const listAllIssuesForRepo = async (state: "open" | "closed" | "all" = "o
   return issuesArr;
 };
 
+export const getIncentivizedUsers = async (issue_number: number) => {
+  const comments = await getAllIssueComments(issue_number);
+  const incentiveComment = comments.filter((comment) => comment.body.startsWith("/comment-incentives"));
+  const parts = incentiveComment[0].body.split("");
```

Comment:
what about if there are multiple `/comment-incentives` comments?

Reply:
what do you suggest?

Reply:
I would iterate from oldest to newest and update accordingly.
For example:
```
/comment-incentives false @test @test2
/comment-incentives true @test @test3
```
then the final configuration should be:
```
@test: true
@test3: true
@test2: false
```